### PR TITLE
Feature/v2 tx card

### PR DESCRIPTION
**Pull request by 0x-shanks**
## Overview
* transaction card component implementation

## TODO
- [x] Responsive support
- [ ] Adjusted width for layout

### CODE REVIEW

File: src/components/Transaction/TagForm.tsx [hunk 1]
```diff
@@ -4,36 +4,73 @@ import {
...
+    <div>
+      <CreatableSelect
+        isMulti
+        defaultValue={defaultVal}
+        onChange={handleChange}
+        options={TagOptions}
+        styles={colourStyles}
+        className={"border-none border-b-gray-400"}
+        menuIsOpen={open}
+        components={{ Input: () => addButton }}
```
Comment (0x-shanks) [inline src/components/Transaction/TagForm.tsx]: Idk why I can't press it when I'm mobile.

File: src/utils/dateUtil.ts [hunk 2]
```diff
@@ -1,11 +1,13 @@
 export const convertTimestampToDateStr = (timestamp: string): string => {
   const d = new Date(Number(timestamp) * 1000);
-  return formatDate(d.toISOString());
-};
-
-const formatDate = (dateStr: string): string => {
-  const d = dateStr.split("-");
-  return `${d[2].split("T")[0]}-${d[1]}-${d[0]}`;
+  d.getMonth();
```
Comment (0x-shanks) [inline src/utils/dateUtil.ts:15]: TODO: It will be affected because it is used a lot in other parts of the system, and I need to define other function.

Comment (0x-shanks) [reply src/utils/dateUtil.ts:15]: fix: ec99ed9

File: src/components/Transaction/TagForm.tsx (hunk 1 above)
Comment (0x-shanks) [reply src/components/Transaction/TagForm.tsx]: rollback : 620bdcb

**Comment by 0x-shanks** [comment]
TODO: The width has to be adjusted to match the layout.

**Review by kitakaze-kan — APPROVED** [review]
LGFM! Thanks!

**Pull request merged**